test/conftest.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
Qua naamgeving van deze file: `conftest.py` klinkt alsof je de configuratie test.
Zou `gridadmin_fixtures.py` beter zijn?

Reply:
Ik heb ze in `conftest.py` gestopt omdat ik dacht dat andere testmodules ook wel gebruik konden maken van deze fixtures: https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions. Maar momenteel worden ze alleen in `test_datasources.py` gebruikt, dus ik kan ze ook daarbij zetten?

Reply:
Ah! `conftest.py` is een specifieke naam waar pytest naar zoekt!
Dat wist ik nog niet :-)